### ISSUE TITLE
Perhaps you can find an event that locates inline with TG1, TG3, and TG6 and plot the waveforms as a function of offset. I would really like to see the moveout of these events and see that the moveout velocity matches your Rayleigh velocity. This would lead me to believe your comments on Pg 5 L133-134. The other reason your event might locate outside (or even in the water, which you do not mention) is because the velocity model is not correct for the type of phase you are inverting.

### ISSUE THREAD

**Issue by staylorofford** [staylorofford/publications#30]

**Comment by staylorofford** [comment]
- I have made half of Fig. 2 a record section of such an event with a moveout velocity of ~1.6 km/s
- I have added to a paragraph to expand on this topic: "An exemplary event in the seismic catalog is presented in Figure~\ref{F2}. The wave velocity, as measured by the difference in arrival times for the event between TG3 and TG4, is $\sim$1.75 km s$^{-1}$ (Figure~\ref{F2}, left). The waveforms recorded at TG3 show a weak wave arriving around 0.25 s and an impulsive, high amplitude wave arriving about 0.05 s later (Figure~\ref{F2}, right). Energy continues to arrive until 0.8 s when the seismogram returns to being noise-dominated (Figure~\ref{F2}, right). The particle motion on the vertical channel consistently lags that of the first horizontal channel by 90 degrees, but is often in-sync with that of the second horizontal channel at this station (Figure~\ref{F2}, right). The seismic catalog contains 1676 similar events with 67\% of event durations between 1--3 seconds. Based on the similarities between events in our catalog and the characteristics of crevassing seismicity, we assume that the source of events in our seismic catalog is crevasse expansion."
- As for events located in the water of elsewhere, I have expanded the section of the results discussing event locations: "Some events locate outside the glacier margins, such as in the proglacial lake or on the adjacent mountains (Figure~\ref{F1}). Such events have been mislocated to the margins of the search grid due to the weak geometric constraints imposed by the station geometry outside the network, and as such are not considered by the authors in any interpretation of the spatial event distribution."
